### ISSUE TITLE
Start location

### ISSUE THREAD

**Issue by BadMagic100** [ArchipelagoMW-HollowKnight/Archipelago.HollowKnight#134]
Implement custom start locations like rando has. Probably needs some server side work to support $STARTLOCATION macros

**Comment by peppidesu** [comment]
Actually, didn't this get implemented with https://github.com/ArchipelagoMW-HollowKnight/Archipelago.HollowKnight/commit/76a6a4c155da4adda4ebf1dd26d18bd1875a77e6?

**Comment by BadMagic100** [comment]
Yeah this is implemented, I've been a bit lazy on the project management
